app/wallet/remote.go

```diff
@@ -47,3 +49,35 @@ func getRemoteUser(url, token string, remoteIP string) (remoteUser, error) {
 
 	return ru, nil
 }
+
+func getRemoteUser(url string, token oauth2.TokenSource, remoteIP string) (remoteUser, error) {
+	op := metrics.StartOperation(opName, "get_remote_user")
+	defer op.End()
+
+	c := lbryinc.NewOauthClient(token, &lbryinc.ClientOpts{
```

Comment:
The two functions are identical save for this part, perhaps we could combine them somehow.

Reply:
Agreed, meant come back to this with a switch parameter or something, forgot. Will push a change up.